test/groupWith.js

```diff
@@ -0,0 +1,31 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('groupWith', function() {
+  var tripleEquals = function(a, b) {
+    return a === b;
+  };
+
+  it('splits the list into groups according to the grouping function', function() {
+    var list1 = [1, 2, 2, 3];
+    assert.deepEqual(R.groupWith(tripleEquals, list1), [[1], [2, 2], [3]]);
+
+    var list2 = [1, 1, 1, 1];
+    assert.deepEqual(R.groupWith(tripleEquals, list2), [[1, 1, 1, 1]]);
+
+    var list3 = [1, 2, 3, 4];
+    assert.deepEqual(R.groupWith(tripleEquals, list3), [[1], [2], [3], [4]]);
+  });
+
+  it('is curried', function() {
+    var groupWithTripleEquals = R.groupWith(tripleEquals);
+    assert.deepEqual(groupWithTripleEquals([1, 1, 2, 2]), [[1, 1], [2, 2]]);
+  });
```

Comment:
I'm in favour of omitting this test. We already have a [test][1] which asserts that every function is curried.


[1]: https://github.com/ramda/ramda/blob/v0.20.0/test/invariants.js#L27-L42